resource/tasks.json

```diff
@@ -298,7 +298,10 @@
         "isAscii": true,
         "text": [],
         "next": ["ClickedCorrectStageOrSwipe", "#self"],
-        "ocrReplace": [["-\\]", "-1"]]
+        "ocrReplace": [
+            ["-\\]", "-1"],
+            ["^.*([A-Z]{2}-EX-\\d+)", "$1"]
+        ]
```

Comment:
I think that's the only modification. Everything else has been reverted.

Reply:
If you think it's good, I'll just commit to dev this :)